doc/source/howtos/virt_sandbox.rst

```diff
@@ -340,59 +423,67 @@ To confirm that the user has been set up correctly, you can try to log in:
 
-   mysql> CREATE USER 'sbuser'@'192.168.70.64' IDENTIFIED BY 'sbpass';
-   Query OK, 0 rows affected (0.01 sec)
+   mysql> GRANT ALL ON *.* TO 'appuser'@'192.168.70.74';
 
-   mysql> GRANT ALL ON *.* TO 'sbuser'@'192.168.70.64';
-   Query OK, 0 rows affected (0.00 sec)
 
-Testing the cluster with sysbench
-=================================
+Testing the cluster with the `sysbench` benchmark tool
+================================================================================
 
-After you set up |PXC| in a sandbox, you can test it using
-`sysbench <https://launchpad.net/sysbench/>`_.
-This example shows how to do it with ``sysbench`` from the EPEL repository.
+After you set up |PXC| in your testing environment, you can test it using
+the ``sysbench`` benchmarking tool.
 
-1. Create a database and a user for ``sysbench``:
+1. Create a database (`sysbenchdb` in this example; you can use a
+   different name) and give full access to it for the `appuser`:
 
    .. code-block:: mysql
 
-      mysql> create database sbtest;
+      mysql> CREATE DATABASE sysbenchdb;
       Query OK, 1 row affected (0.01 sec)
 
-      mysql> grant all on sbtest.* to 'sbtest'@'%' identified by 'sbpass';
+      mysql> GRANT ALL ON sysbenchdb.* TO 'appuser'@'%' IDENTIFIED BY '$3kRetp@$sW0rd';
       Query OK, 0 rows affected (0.00 sec)
 
-      mysql> flush privileges;
+      mysql> FLUSH PRIVILEGES;
       Query OK, 0 rows affected (0.00 sec)
 
-#. Populate the table with data for the benchmark:
+#. Populate the table with data for the benchmark. Note that you
+   should pass the database you have created as the value of the
+   `--mysql-db` parameter, and the name of the user who has full
+   access to this database as the value of the `--mysql-user`
+   parameter:
 
    .. code-block:: bash
 
-      sysbench --test=oltp --db-driver=mysql --mysql-engine-trx=yes \
-      --mysql-table-engine=innodb --mysql-host=127.0.0.1 --mysql-port=3307 \
-      --mysql-user=sbtest --mysql-password=sbpass --oltp-table-size=10000 prepare
+      $ sysbench /usr/share/sysbench/oltp_insert.lua --mysql-db=sysbenchdb \
+      --mysql-host=127.0.0.1 --mysql-port=6033 --mysql-user=appuser \
+      --mysql-password=$3kRetp@$sW0rd --db-driver=mysql --threads=10 --tables=10 \
+      --table-size=1000 prepare
 
-#. Run the benchmark on port 3307:
+#. Run the benchmark on port 6033:
 
    .. code-block:: bash
 
-      sysbench --test=oltp --db-driver=mysql --mysql-engine-trx=yes \
-      --mysql-table-engine=innodb --mysql-host=127.0.0.1 --mysql-port=3307 \
-      --mysql-user=sbtest --mysql-password=sbpass --oltp-table-size=10000 \
-      --num-threads=8 run
+      $ sysbench /usr/share/sysbench/oltp_read_write.lua --mysql-db=sysbenchdb \
+      --mysql-host=127.0.0.1 --mysql-port=6033 --mysql-user=appuser \
+      --mysql-password=$3kRetp@$sW0rd --db-driver=mysql --threads=10 --tables=10 \
+      --table-size=1000 --time=100 --report-interval=10 run
```

Comment:
sysbench by default encapsulates all the queries within a transaction (https://github.com/akopytov/sysbench/blob/master/src/lua/oltp_common.lua#L64), which means it will redirect all the traffic to the master and the above command is not testing load balancing. I would suggest we add `--skip_trx=true` as part of the command